tests/functional/centos/7/cluster/vagrant_variables.yml

```diff
@@ -10,7 +10,7 @@ mds_vms: 1
 rgw_vms: 1
 nfs_vms: 0
 rbd_mirror_vms: 0
-client_vms: 0
+client_vms: 1
```

Comment:
@leseb No, I mean you need to add a section like this to your hosts file:

```
[clients]
client0
```

Like I've done here: https://github.com/ceph/ceph-ansible/pull/1236/files#diff-b0b857e9bebef348b0c2b15e96769ab6R15

My other comment was about testing the inverse of what you've done for ``centos7_cluster`` in ``xenial_cluster``. Meaning, add a client vm, add it to the hosts file and then do not set ``user_config`` or ``openstack_config`` in the group_vars for ``xenial_cluster``.